app/assets/stylesheets/preact/article-form.scss

```diff
@@ -173,8 +173,8 @@
     margin-bottom: 0px;
     margin-bottom: 1px;
     @media screen and (min-width: 600px) {
-      height: 32px;
-      margin-top: 2px;
+      height: 43px;
```

Comment:
Not by my computer, but these changes look good in Chrome, FF and Safari?

Reply:
Yep